src/dune-project

```diff
@@ -1,2 +1 @@
 (lang dune 3.1)
-(implicit_transitive_deps false)
```

Comment:
I'm pretty sure we still want this, and on general this is not the PR to remove it in.

Reply:
Good catch! Sorry, not sure how it got here.